src/pipeline-server/docker-compose.pipeline-server.yml

```diff
@@ -146,16 +138,8 @@ services:
       - /run:uid=1999,gid=1999
     security_opt:
       - no-new-privileges
-    environment:
-      MODE: "EVA"
-      http_proxy: ${http_proxy}
-      https_proxy: ${https_proxy}
-      no_proxy: ${no_proxy}
```

Comment:
Same as above, it is recommended to take the proxy settings from the host system.